components/sp-solutions/org.wso2.carbon.sp.solutions.apim.analytics/components/org.wso2.sp.extension.siddhi.io.mgwfile/pom.xml

```diff
@@ -0,0 +1,94 @@
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.wso2.analytics.solutions</groupId>
+        <artifactId>org.wso2.carbon.sp.solutions.apim.analytics</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>org.wso2.sp.extension.siddhi.io.mgwfile</artifactId>
+    <packaging>bundle</packaging>
+    <name>WSO2 SP - Apim Analytics Micro Gateway File</name>
+    <description>Apim Analytics Micro Gateway Component</description>
+    <url>http://wso2.org</url>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.siddhi</groupId>
+            <artifactId>siddhi-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.siddhi</groupId>
+            <artifactId>siddhi-annotations</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.siddhi</groupId>
+            <artifactId>siddhi-query-compiler</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.siddhi</groupId>
+            <artifactId>siddhi-query-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.log4j.wso2</groupId>
+            <artifactId>log4j</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.analytics-common</groupId>
+            <artifactId>org.wso2.carbon.databridge.core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.extension.siddhi.map.json</groupId>
+            <artifactId>siddhi-map-json</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.extension.siddhi.map.wso2event</groupId>
+            <artifactId>siddhi-map-wso2event</artifactId>
+        </dependency>
+    </dependencies>
+
+
+    <properties>
+        <import.package>
+            *;resolution:=optional
+        </import.package>
+        <export.package>
+            org.wso2.extension.siddhi.io.mgwfile.*
+        </export.package>
+        <Include-Resource>
+            META-INF=target/classes/META-INF
+        </Include-Resource>
+    </properties>
+
+</project>
```

Comment:
add new line EOF

Reply:
Fixed in a21aa920ceb5fdbd3f34fa9dddb75b5630876048 of https://github.com/wso2/analytics-apim.